doc/user/zebra.rst

```diff
@@ -155,6 +155,12 @@ Standard Commands
    Set description for the interface.
 
 
+.. clicmd:: mpls
```

Comment:
We need to update documentation too (`mpls enable`). Also, it might be worth mentioning it works with "newer" kernels? Like CentOS 7.x or so won't work, right?

Reply:
fixed